core/spec/lib/spree/app_configuration_spec.rb

```diff
@@ -157,56 +157,4 @@ class DummyClass; end;
     expect(prefs.admin_vat_location.state_id).to eq(nil)
     expect(prefs.admin_vat_location.country_id).to eq(nil)
   end
-
-  it 'has default Event adapter' do
-    expect(prefs.events.adapter).to eq Spree::Event::Adapters::ActiveSupportNotifications
-  end
-
-  context 'mails_from' do
-    it 'is deprecated' do
-      expect(Spree::Deprecation).to receive(:warn).with(/Solidus doesn't use `Spree::Config.mails_from`/)
-
-      prefs.mails_from=('solidus@example.com')
-    end
-
-    it "still sets the value so that consumers aren't broken" do
-      Spree::Deprecation.silence do
-        prefs.mails_from=('solidus@example.com')
-      end
-
-      expect(prefs.mails_from).to eq('solidus@example.com')
-    end
-  end
-
-  context 'extra_taxon_validations=' do
-    it 'is deprecated' do
-      expect(Spree::Deprecation).to receive(:warn).with(/Solidus will remove `Spree::Config.extra_taxon_validations`/)
-
-      prefs.extra_taxon_validations=(false)
-    end
-
-    it "still sets the value so that consumers aren't broken" do
-      Spree::Deprecation.silence do
-        prefs.extra_taxon_validations=(false)
-      end
-
-      expect(prefs.extra_taxon_validations).to eq(false)
-    end
-  end
-
-  context 'extra_taxonomy_validations=' do
-    it 'is deprecated' do
-      expect(Spree::Deprecation).to receive(:warn).with(/Solidus will remove `Spree::Config.extra_taxonomy_validations`/)
-
-      prefs.extra_taxonomy_validations=(false)
-    end
-
-    it "still sets the value so that consumers aren't broken" do
-      Spree::Deprecation.silence do
-        prefs.extra_taxonomy_validations=(false)
-      end
-
-      expect(prefs.extra_taxonomy_validations).to eq(false)
-    end
-  end
```

Comment:
This diff leaked from another deprecation.

Reply:
Oh! Thanks for spotting this, not sure how it slipped into this PR, going to address it.